source/tutorials/php.rst

```diff
@@ -1,193 +1,251 @@
+   To enable an extension, you must install it, add it to the
+   :file:`my-php.ini` file, and restart the :file:`php-fpm` service. However,
+   some extensions may have configuration options, which will be documented by
+   the extension maintainer. Add the options you need to the
+   :file:`/etc/php.d/my-php.ini` file as described in the extension's
+   documentation. Be sure to restart :file:`php-fpm` after changing the file.
 
 Configure PHP-FPM
 *****************
 
-The PHP-FPM configuration file is separate from the :file:`php.ini` file used by PHP. |CL| installs the default :file:`php-fpm.conf` file in :file:`/usr/share/defaults/php` and this file will be overwritten with its default values during each software update. However, PHP-FPM requires that the configuration file exist in that location, and by design will not read configuration options from a different path.
-
-One solution to changing PHP-FPM configuration options in |CL| is to manually override the php-fpm.service unit in systemd to pass an explicit location to a custom :file:`php-fpm.conf` file.
+The PHP-FPM configuration file is separate from the :file:`php.ini` file used
+by PHP, however, it has a similar default path restriction. Follow the steps
+below to configure PHP-FPM.
 
-#. Copy the :file:`/usr/share/defaults/php/php-fpm.conf` file to the :file:`/etc/php.d`
 
-   .. code-block:: bash
+#.  Copy the :file:`/usr/share/defaults/php/php-fpm.conf` file to the
+    :file:`/etc/php.d` file:
 
-      sudo cp /usr/share/defaults/php/php-fpm.conf /etc/php.d/php-fpm.conf
+    .. code-block:: bash
 
-#. Make changes to the :file:`php-fpm.conf` file as needed. The `FPM documentation`_ has detail on the configuration options available to PHP-FPM.
+       sudo cp /usr/share/defaults/php/php-fpm.conf /etc/php.d/php-fpm.conf
 
+#.  Make changes to the :file:`php-fpm.conf` file as needed. The
+    `FPM documentation`_ has detail on the configuration options available
+    to PHP-FPM.
 
-#. Edit the systemd service unit file
+#.  Edit the systemd service unit file:
 
-   .. code-block:: bash
+    .. code-block:: bash
 
-      sudo systemctl edit --full php-fpm.service
+       sudo systemctl edit --full php-fpm.service
 
-   This will open the php-fpm.service file for systemd in your editor.  Change the  :command:`ExecStart` configuration to add the :command:`--fpm-config` option to point to the custom location.
+    This opens the php-fpm.service file for systemd in your editor.
 
-   .. code-block:: bash
+#.  Change the :command:`ExecStart` configuration to add the
+    :command:`--fpm-config` option to point to the custom location:
 
-      sudo systemctl edit --full php-fpm.service
+    .. code-block:: bash
 
-   .. code-block:: console
+       sudo systemctl edit --full php-fpm.service
```

Comment:
@bd-dean  This command is the same as the prior step. Can you check your notes for the correct command?

Reply:
@bd-dean Please let use know if this is an error. Thanks.

Reply:
the command is redundant in this place & can be deleted. Or, the block starting at line #165 can be deleted.

Reply:
Agreed.